staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation.go

```diff
@@ -63,21 +64,74 @@ func CheckAllowPrivilegeEscalation() Check {
 
 func allowPrivilegeEscalation_1_8(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) CheckResult {
```

Comment:
```suggestion
type option struct {
  withErrList bool
}

type Option = func(*option)

func WithErrList() Option {
  return func(opt *option) {
    opt.withErrList = true
  }
}

func allowPrivilegeEscalation_1_8(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec, options ...Option) CheckResult {
  var opt option
  for _, o := range options {
    o(&opt)
  }
```

Something along those lines.